lib/client/native_client.dart

```diff
@@ -142,11 +143,19 @@ class Config {
 class NativeClient {
   late final DynamicLibrary _wormholeWilliamLib;
   late final DynamicLibrary _asyncCallbackLib;
+
   late final Config config;
 
   NativeClient({Config? config}) {
```

Comment:
@ewanas can we make error handling here in case we fail to load library? For example, if I run iOS app on amd64 arch and if fails to load static libraries, receive or sender is just as waiting.. Probably we should throw error and display something to user as technical error.

Reply:
Because we rely on `executable()` as the source of symbols here, the case of "could not find the library" isn't possible. Right now symbols are looked up when a client function needs them. Maybe the constructor for `NativeClient` can [lookup](https://api.dart.dev/stable/2.18.6/dart-ffi/DynamicLibrary/lookup.html) or [check](https://api.dart.dev/stable/2.18.6/dart-ffi/DynamicLibrary/providesSymbol.html) if all needed symbols beforehand and fail with an error.